include/ocpn_plugin.h

```diff
@@ -558,6 +558,22 @@ class DECL_EXP opencpn_plugin_116 : public opencpn_plugin_115
 
 };
 
+class DECL_EXP opencpn_plugin_117 : public opencpn_plugin_116
```

Comment:
Why not just add the `SemanticVersion   GetVersion();` to the new plugin API here, avoiding the three functions Path/Pre/Build functions? (and deprecate the old functions)? The current approach might be nicer for existing plugins.

Reply:
Perhaps... but I basically hesitate to add this somewhat complex datatype to the API in attempt to keep it less messy. It's not an easy choice and could need  some nights sleep.

Reply:
OK, been sleeping on this. I think it's better to keep the API as simple as possible w r t used types.  After all, the version data is not consumed in the plugins, they just provide it. 

That said:  it's not an easy call and a good question  to raise.

Reply:
Makes sense, it is good as it is.